src/compile/invalid/normalizeInvalidDataMode.ts

```diff
@@ -0,0 +1,15 @@
+import {MarkInvalidDataMode} from '../../invalid';
+
+type NormalizedMarkInvalidDataMode = Exclude<MarkInvalidDataMode, 'break-paths-and-keep-path-domains'>;
+
+export function normalizeInvalidDataMode(
+  mode: MarkInvalidDataMode | null | undefined,
+  {isPath}: {isPath: boolean}
+): NormalizedMarkInvalidDataMode {
+  if (mode === undefined || mode === 'break-paths-and-keep-path-domains') {
```

Comment:
should we have a const DefaultMarkInvalidDataMode = 'break-paths-and-keep-path-domains' and use it here?

Reply:
that seems unnecessary? 

We use these undefined check everywhere in Vega-Lite.  `undefined` normally means default already.